IBAnimatable/AnimatableBarButtonItem.swift

```diff
@@ -45,6 +45,6 @@ open class AnimatableBarButtonItem: UIBarButtonItem, BarButtonItemDesignable, An
   // MARK: - Private
   fileprivate func configureInspectableProperties() {
 //    configureAnimatableProperties()
```

Comment:
Can you remove that line? I know it's not related to your PR, but it still could be a good move!